### DISCUSSION TITLE
Border when using {%table}

### DISCUSSION THREAD

**Discussion by rocketbumba** [markdoc/markdoc#491] [Q&A] [answered]
When i use %table, it seem like no render border on my HTML. But when i use sandbox of Markdoc, it gen border. Do anyone know the answer?

**Comment by mfix-stripe** [comment, marked as answer]
@rocketbumba this probably just depends on the CSS you are using. Our Sandbox uses particular CSS rules that we have pre-applied, manually, to make tables look a certain way across our site.
